dbms/src/Storages/DeltaMerge/Filter/And.h

```diff
@@ -44,7 +44,7 @@ class And : public LogicalOp
         return res;
     }
 
-    ColumnRangePtr buildSets(const LocalIndexInfosSnapshot & index_info) override
+    ColumnRangePtr buildSets(const google::protobuf::RepeatedPtrField<tipb::ColumnarIndexInfo> & index_info) override
```

Comment:
Interesting, does this actually only accept InvertedIndexes?

Reply:
No, maybe there is no need to copy them to a new container.